gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
> require_relative usually requires you to maintain the same folder structure so once you move your code around, you need to make those changes. 

nope... that is why we require by gem name. Only if the gem name changes will this statement be impacted.

> require_relative '../lib/earlgrey'